pkg/generate/ack/controller.go

```diff
@@ -22,6 +22,7 @@ import (
 	ackgenconfig "github.com/aws-controllers-k8s/code-generator/pkg/generate/config"
 	"github.com/aws-controllers-k8s/code-generator/pkg/generate/templateset"
 	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
+	ackoperations "github.com/aws-controllers-k8s/code-generator/pkg/operations"
```

Comment:
The import aliasing in ACK seems to be a bit random. Some places use `ackmodel` others import model without an alias. I have just stuck to whatever the file has as convention. I don't think `ackmodel` is necessary, because we are clearly in the `ack` project, so `model` should be sufficient.